### ISSUE TITLE
File Confirmation Statement Online 17 June 2017

### ISSUE THREAD

**Issue by iteles** [dwyl/hq#288]
It would seem this has replaced the annual return and is now 2 weeks overdue 😳 

https://www.1stformations.co.uk/blog/confirmation-statement-explained/

**Comment by markwilliamfirth** [comment]
#262 needs to be completed prior to this

**Comment by markwilliamfirth** [comment]
Companies House have been informed.

The only thing this may affect is the company credit rating

#262 required in order to progress @nelsonic @iteles

**Comment by iteles** [comment]
I've just been reading https://www.gov.uk/government/publications/confirmation-statement/confirmation-statement and it implies to me that #262 is _not_ a prerequisite but rather more complete information of the annual confirmation (which is the actual thing that should have been filed in April). 

Am I misreading this?

**Comment by markwilliamfirth** [comment]
@iteles you have to make all the separate filings first, the confirmation statement follows all of the separate filings - it confirms everything that occurred during the year, not just these filings we have to do now. If the confirmation statement goes in without us doing these filings first, it'll be rejected because it doesn't match the company history.

**Comment by markwilliamfirth** [comment]
Closing in favour of #262

**Comment by iteles** [comment]
@markwilliamfirth If you're closing this issue, can we at least make the Annual confirmation statement a checkbox in the one you're closing it in favour of? 
It's not even listed there at the moment and that's making me a bit nervous 😅

**Comment by markwilliamfirth** [comment]
It is there - third from the bottom :)

**Comment by markwilliamfirth** [comment]
Reopening this issue as we'll be filing this separately online to the paper documents. The reason for this is because it is quicker and cheaper (only £13) to file online.

The next steps is to send off the docs https://github.com/dwyl/hq/issues/262 and wait a few days for the changes to appear on beta.companieshouse.gov.uk and then file the confirmation statement online before the 7th June Deadline

**Comment by markwilliamfirth** [comment]
This has been done
